datafusion/results/partitioned.json

```diff
@@ -1,55 +1,55 @@
 {
     "system": "DataFusion (Parquet, partitioned)",
-    "date": "2024-03-29",
+    "date": "2024-04-18",
     "machine": "c6a.4xlarge, 500gb gp2",
     "cluster_size": 1,
-    "comment": "v45.0.0 (26058ac)",
+    "comment": "v46.0.0 (d5ca830)",
     "tags": ["Rust", "column-oriented", "embedded", "stateless"],
     "load_time": 0,
-    "data_size": 14779976446,
```

Comment:
I think this was just copy-pasted from the single file, and the data isn't actually the same size.